src/components/02_atoms/PageTitle/PageTitle.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withSideEffect from 'react-side-effect';
+import { css } from 'emotion';
+
+import Typography from '@material-ui/core/Typography';
+
+const styles = {
+  title: css`
+    margin-bottom: 15px;
+    margin-left: 3px;
+  `,
+};
+
+const PageTitle = ({ children }) => (
+  <Typography variant="headline" classes={{ root: styles.title }}>
+    {children}
+  </Typography>
+);
+
+PageTitle.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+function reducePropsToState(propsList) {
```

Comment:
What is the reason to not use ```const = ```functions like everywhere else?

Reply:
Good point! 👍 This was leftover from a copy and paste from the example 🙂